static-site/src/sections/group-members-page.tsx

```diff
@@ -100,7 +100,8 @@ const MembersTableContainer = styled.div`
 `;
 
 const MembersTable = ({ members }: { members: GroupMember[]}) => {
-  const sortedMembers = members.toSorted((a, b) => a.username.localeCompare(b.username));
+  const sortedMembers = [...members];
+  sortedMembers.sort((a, b) => a.username.localeCompare(b.username));
```

Comment:
_non-blocking, purely style_

```suggestion
  const sortedMembers = [...members].sort((a, b) => a.username.localeCompare(b.username));
```

Reply:
Ah, right Javascript's `sort()` returns the reference to the array! Mixing up with Python's `sort()` which returns `None`

Reply:
Updated in [force-push](https://github.com/nextstrain/nextstrain.org/compare/572934557853e33b81d28dab5557ef783f13982c..c7d9a0b2aa9c226f7b5096a26b96eb7334125a4e)